set.mm

```diff
@@ -422480,6 +422481,16 @@ Class symbols for functions (e.g., ` abs ` , ` sin ` )
        ` ( V iEdg E ) ` for the edges of a graph ` G = <. V , E >. ` .
        </li>
 
+       <li><b>LaTeX definitions.</b>
+       Each token has a "LaTeX definition" which is used by metamath.exe to
+       output tex files.  When writing LaTeX definitions, contributors should
+       favor simplicity over perfection of the display, and should only use
+       core LaTeX symbols or symbols from the AMS packages (amssymb, amsmath)
+       and the mathtools package (exceptions to this should be discussed).  A
+       useful resource is <a href = https://www.ctan.org/pkg/comprehensive>
```

Comment:
Please don't use:

~~~~
 <a href = https://www.ctan.org/pkg/comprehensive>
~~~~

In particular, no spaces around the "=", and enclose the value in double-quotes, like thisl

~~~~
<a href="https://www.ctan.org/pkg/comprehensive">
~~~~

I believe the [space-around-equals will *work* in typical browsers](https://www.w3schools.com/html/html5_syntax.asp) but it's not considered good practice, and almost certainly some processors won't handle it correctly.